hazelcast/pom.xml

```diff
@@ -748,5 +748,12 @@
             <version>${project.parent.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.hazelcast.jet</groupId>
+            <artifactId>hazelcast-jet-avro</artifactId>
+            <version>5.1.1</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
This creates a circular dependency between `hazelcast` and `hazelcast-jet-avro` modules, that's generally bad thing leading to flaky builds.